dev/landing-pages/this-is-big.js

```diff
@@ -0,0 +1,13 @@
+function homePageMajorsList() {
```

Comment:
I probably should have just renamed this to topMajorsToggle and the init function is not necessary.